packages/cli/src/lib/local/run-foreman.js

```diff
@@ -227,7 +229,9 @@ program
         var conf = {}
         conf.number = i
 
+        // eslint-disable-next-line block-scoped-var
         for (_ in c) {
+          // eslint-disable-next-line block-scoped-var
           conf[_] = c[_]
         }
```

Comment:
Can this just be `const conf = {...c, number: i}` ?

Reply:
yes, that looks like it would work. I was trying to avoid refactoring as much as possible.